### PR TITLE
Rewrite tutorials and templates

### PR DESCRIPTION
**Pull request by igordmn**
- Remove `templates` folder (in fact it is renamed and hidden, because we need it for CI). We will point to the separate templates repos
- Remove Desktop. Getting Started. It is described in the template itself
- Remove a separate Android doc, it only repeats information from other places
- Rename Compose for Web to Compose HTML Library
- Keep Getting started with Compose HTML library
- Remove Wizard from Getting Started with Compose HTML Library

### CODE REVIEW

File: tutorials/HTML/Getting_Started/README.md [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
 * JDK 11 or later
 * IntelliJ IDEA Community Edition or Ultimate Edition 2020.2 or later (you can use other editors, but for this tutorial we assume you are using IntelliJ IDEA)
 
-
 ## Creating a new project
 
-You can create the project by [downloading the template here](https://github.com/JetBrains/compose-multiplatform/tree/master/templates/web-template) or using the Project Wizard in IDEA:
-
-<img alt="Create new project" src="wizard.png" height="500" />
+You can create the project by [downloading the template here](https://github.com/JetBrains/compose-multiplatform-html-library-template).
```
Comment (igordmn) [inline tutorials/HTML/Getting_Started/README.md:13]: I suggest to keep https://github.com/JetBrains/compose-multiplatform-web-template for now, and rename it to `compose-multiplatform-html-library-template`

**Comment by igordmn** [comment]
Will rework examples README in a separate PR

**Review by eymar — APPROVED** [review]

**Comment by malliaridis** [comment]
@igordmn can you have a look at https://github.com/JetBrains/compose-multiplatform/pull/2933? Changes may be related to this PR.

**Comment by igordmn** [comment]
> @igordmn can you have a look at https://github.com/JetBrains/compose-multiplatform/pull/2933? Changes may be related to this PR.

Thanks! I will look at it later.

This tutorial isn't touched by this PR.

**Comment by eymar** [comment]
https://github.com/JetBrains/compose-multiplatform/pull/3037 - this should fix the templates checker according to changes in the current PR.

File: tutorials/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+* [Navigation](Navigation)
+* [Accessibility](Accessibility)
+* [Building a native distribution](Native_distributions_and_local_execution)
+
+## Web
+* [Getting started](https://kotl.in/wasm-compose-example)
+
+## General
+
+See [Foundation](https://developer.android.com/jetpack/compose/documentation#core) and [Design](https://developer.android.com/jetpack/compose/documentation#design) tutorials from Google. They were originally written for Android, but most of information applies for the other targets as well.
```
Comment (katia-energizer) [inline tutorials/README.md:27]:
```suggestion
Google's [Foundation](https://developer.android.com/jetpack/compose/documentation#core) and [Design](https://developer.android.com/jetpack/compose/documentation#design) tutorials were originally written for Android but can be used as a reference for other targets.
```

File: tutorials/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Libraries
+
+### HTML
+Compose HTML Library allows you to build reactive user interfaces for the web in Kotlin, using the concepts and APIs of Compose to express the state, behaviour, and logic of your application.
```
Comment (katia-energizer) [inline tutorials/README.md:32]:
```suggestion
The Compose HTML library allows you to build reactive user interfaces for the web in Kotlin using the concepts and APIs of Compose to express the state, behavior, and logic of your application.
```

Comment (katia-energizer) [reply tutorials/README.md:32]: Please replace in other places `Compose HTML Library` with `the Compose HTML library`. We should capitalize only the name of the library and use an article.

File: tutorials/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+HTML Library provides multiple ways of declaring user interfaces in Kotlin code, allowing you to have full control over your website layout with a declarative HTML/CSS API.
```
Comment (katia-energizer) [inline tutorials/README.md:34]:
```suggestion
The HTML library provides multiple ways of declaring user interfaces in Kotlin with an ability to fully control your website layout with a declarative HTML/CSS API.
```

File: tutorials/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+> Note that this library currently supports only Kotlin JS, not Kotlin/Wasm
```
Comment (katia-energizer) [inline tutorials/README.md:36]:
```suggestion
> Note that this library doesn't Kotlin/Wasm yet.
```

Comment (katia-energizer) [reply tutorials/README.md:36]: I would avoid mentioning Kotlin/JS.

**Review by katia-energizer — CHANGES_REQUESTED** [review]
Left comments.

File: tutorials/HTML/Getting_Started/README.md (hunk 1 above)
Comment (katia-energizer) [reply tutorials/HTML/Getting_Started/README.md:13]: I don't mind for now.

File: tutorials/README.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+* [Top level windows management](Window_API_new)
+* [Menu, tray, notifications](Tray_Notifications_MenuBar_new)
+* [Keyboard support](Keyboard)
+* [Tab focus navigation](Tab_Navigation)
+* [Swing interoperability](Swing_Integration)
+* [Navigation](Navigation)
+* [Accessibility](Accessibility)
+* [Building a native distribution](Native_distributions_and_local_execution)
+
+## Web
```
[the remark this answers is not in the document]
Comment (igordmn) [reply tutorials/README.md:22]: If we have only HTML on this page, people can mistake it with the web target.

Maybe we can write something like this instead?

> The web target is experimental. You can find a demo [here].

P.S. We have `Get started with Compose for Web` on the main page as well

File: tutorials/HTML/Getting_Started/README.md (hunk 1 above)
Comment (igordmn) [reply tutorials/HTML/Getting_Started/README.md:13]: Renamed the repo

File: tutorials/README.md (hunk 2 above)
Comment (igordmn) [reply tutorials/README.md]: Done

File: tutorials/README.md (hunk 3 above)
Comment (igordmn) [reply tutorials/README.md]: Done

File: tutorials/README.md (hunk 4 above)
Comment (igordmn) [reply tutorials/README.md]: Done

**Pull request merged**